packages/edit-site/src/components/dataviews/text-filter.js

```diff
@@ -10,12 +10,12 @@ import { SearchControl } from '@wordpress/components';
  */
 import useDebouncedInput from '../../utils/use-debounced-input';
 
-export default function TextFilter( { filter, view, onChangeView } ) {
+export default function TextFilter( { label, view, onChangeView } ) {
```

Comment:
This should probably be renamed `Search`  or something like that as it's not generic anymore.

Reply:
Agreed, though didn't want to do it in this PR to make reviews easier.

Reply:
https://github.com/WordPress/gutenberg/pull/55731